pkg/crypto/clef/clef.go

```diff
@@ -109,3 +118,14 @@ func (c *clefSigner) SignTx(transaction *types.Transaction) (*types.Transaction,
 func (c *clefSigner) EthereumAddress() (common.Address, error) {
 	return c.account.Address, nil
 }
+
+// SignTypedData signs data according to eip712
+func (c *clefSigner) SignTypedData(typedData *eip712.TypedData) ([]byte, error) {
```

Comment:
maybe `SignTyped` is enough?

Reply:
Possibly. But this signing method is called `signTypedData` both on an rpc level and in all other libraries I saw.